packages/wallet-frontend/test/test-lib-wallet.js

```diff
@@ -12,8 +12,8 @@ import fakeVatAdmin from '@agoric/zoe/test/unitTests/contracts/fakeVatAdmin';
 import { makeRegistrar } from '@agoric/registrar';
 
 import { E } from '@agoric/eventual-send';
-import { makeWallet } from '../../lib/ag-solo/vats/lib-wallet';
-import { makeBoard } from '../../lib/ag-solo/vats/lib-board';
+import { makeWallet } from '../lib/lib-wallet';
+import { makeBoard } from '@agoric/cosmic-swingset/lib/ag-solo/vats/lib-board';
```

Comment:
I hope we can eventually rearrange things to not require deep-imports like this, especially given how we want to break cosmic-swingset up into a chain-side package and a solo-side package. But it's fine for now.

Reply:
Yeah, the board needs to be its own package.  Not quite ready for that yet, though.